scintillometry/__init__.py

```diff
@@ -27,3 +27,5 @@ class UnsupportedPythonError(Exception):
 if not _ASTROPY_SETUP_:
     # For egg_info test builds to pass, put package imports here.
     pass
+
+from .channelize import ChannelizeTask
```

Comment:
Let's not do the top-level imports yet - if we go that route that means that if you import anything from `scintillometry` you automatically import everything, which may not be very useful later on.

Reply:
Sure, removed.